import org.apache.hadoop.fs.PathIOException;
public class FunctionUse12899 {
public void funcUse() {
PathIOException pathioexception = new PathIOException();
pathioexception.setTargetPath(null);
}
}